backend/ohq/models.py

```diff
@@ -198,6 +198,8 @@ class Queue(models.Model):
     question_template = models.TextField(blank=True, default="")
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
     archived = models.BooleanField(default=False)
+    is_pin_enabled = models.BooleanField(default=False)
+    pin = models.CharField(max_length=5, default="", blank=True, null=True)
```

Comment:
Looking at this again, I don't think we actually need the `default=""` here since we added that this field can be null, meaning pin would just be set to null in cases that it isn't provided